DATA_FORMAT.md

```diff
@@ -0,0 +1,330 @@
+    "tracks": "tracks.json",
+    "times": "times.json",
+    "centroids": "centroids.json",
+    "bounds": "bounds.json"
+}
+```
+
+---
+
+</details>
+
+### 1. Tracks
+
+Every segmented object in each time step has a **track ID**, an integer identifier that's unique across all time steps. To recognize a single track across multiple frames, these track IDs must be grouped together with a single **track number**.
+
+A **track JSON file** consists of a JSON object with a `data` array, where for each track ID `i`, `data[i]` is the track number that track ID is part of.
+
+```
+--track.json--
+{
+    "data": [
+        <track number for id 0>,
+        <track number for id 1>,
+        <track number for id 2>,
+        ...
+    ]
+}
+```
+
+<details>
+<summary><b>[Show me an example!]</b></summary>
+
+---
+
+For example, if there were the following two tracks in some dataset, the track file might look something like this.
+
+| Track # | Track IDs |
+| ------- | --------- |
+| 1       | 0, 1, 4   |
+| 2       | 2, 3, 5   |
+
+```
+--track.json--
+{
+    "data": [
+        1, // 0
+        1, // 1
+        2, // 2
+        2, // 3
+        1, // 4
+        2  // 5
+    ]
+}
+```
+
+---
+
+</details>
+
+### 2. Times
+
+The times JSON is similar to the tracks JSON. It also contains a `data` array that maps from track IDs to the frame number that they appear on.
+
+```
+--times.json--
+{
+    "data": [
+        <frame number for track id 0>,
+        <frame number for track id 1>,
+        <frame number for track id 2>,
+        ...
+    ]
+}
+```
+
+### 3. Frames
+
+_Example frame:_
+![](./documentation/frame_0.png)
+_Each unique color in this frame is a different track ID._
+
+**Frames** are image textures that store the track IDs for each time step in the time series. Each pixel in the image can encode a single track ID in its RGB value (`= R + G*256 + B*256*256`). Note that there is currently no way to indicate overlaps in the data-- your script will need to flatten 3D data and segmentations.
+
+There should be one frame for every time step in the time series, and they must all be listed in order in the **manifest** file.
+
+### 4. Features
+
+Datasets can contain any number of features, which are a numeric value assigned to each track ID in the dataset. Each feature file usually corresponds to a single column of data.
+
+Features must also provide a `min` and `max` range property.
+
+```
+--feature1.json--
+{
+    "data": [
+        <feature value for track id 0>,
+        <feature value for track id 1>,
+        <feature value for track id 2>,
+        ...
+    ],
+    "min": <min value for all features>,
+    "max": <max value for all features>
+}
+```
+
+### 5. Centroids
+
+The centroids file defines the center of each track ID in the dataset. For each index `i`, the coordinates are `(x: data[2i], y: data[2i + 1])`.
+Coordinates are defined in pixels in the frame, where the upper left corner of the frame is (0, 0).
```

Comment:
Even though the example should be clear, I might just also state that the centroids must follow the same format as a feature, except that the x y coordinates must occur sequentially so the list of values will be twice as long as for a feature.